libuserhelper.cpp

```diff
@@ -110,6 +110,37 @@ bool LibUserHelper::addUserToGroup(const QString &user, const QString &group)
     return rv;
 }
 
+bool LibUserHelper::removeUserFromGroup(const QString &user, const QString &group)
+{
+    struct lu_error *error = nullptr;
+    struct lu_context *context = lu_start(NULL, lu_user, NULL, NULL, NULL, NULL, &error);
+    if (!context) {
+        qCWarning(lcSUM) << "Error creating context";
+        return false;
+    }
+
+    bool rv = true;
+    struct lu_ent *ent = lu_ent_new();
+    if (lu_group_lookup_name(context, group.toLocal8Bit(), ent, &error)) {
+        GValue *value = g_slice_new0(GValue);
```

Comment:
Who frees this `GValue` and how? Have you run this code under valgrind? If not, I suggest to do so.

Reply:
Added frees.